web/src/components/ui/card.stories.tsx

```
@@ -0,0 +1,93 @@
+  }
+};
+
+export const WithError: Story = {
+  render: (args) => <Input {...args} />,
+  args: {
+    className: 'border-red-500 focus-visible:ring-red-500',
+    placeholder: 'Error state...'
+  }
+};
+
+export const Number: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)